content/glossary/_index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Glossary
+toc: true
+# View.
+#   1 = List
+#   2 = Compact
+#   3 = Card
+#   4 = Citation
+view: 1
+
+# Optional header image (relative to `static/media/` folder).
+header:
+  caption: ""
+  image: "resources.jpg"
+---
+
```

Comment:
```
{{% callout note %}}
A Markdown callout is useful for displaying notices, hints, or definitions to your readers.
{{% /callout %}}
```
That's what you want to use @flavioazevedo

Reply:
Thanks!

Reply:
```suggestion
{{% callout note %}}
This is a working in progress version of the Glossary which are (probably) filled with mistakes. This uses an older, non-reviewed version of the Glosssary. We are trying to see whats the best way to show information and terms.
{{% /callout %}}

```